### ISSUE TITLE
join polylines

### ISSUE THREAD

**Issue by chris3522** [mbloch/mapshaper#561]
Hello,

I was wondering if it was possible to keep the lines of a layer if they are partly superimposable on another layer.
On the red layer road1 we see a segment of black road 80_D1015 and on the gray layer road2 we see the same longer black road but badly drawn but with intersection points.
Can we create a correspondence field between the two on one of the layers? For example on layer 1 add to segment 80_D1015 the INSEE field of the underlying road?

Once again many thanks
![roads1](https://user-images.githubusercontent.com/60651125/194341262-3a4bbe57-e60e-48ac-9e12-0a2b25a38dbe.png)
![roads2](https://user-images.githubusercontent.com/60651125/194341272-6b03b172-87db-492c-83ec-cac41ad1dba5.png)

**Comment by chris3522** [comment]
I try to approach the problem from another angle. Maybe it's possible to create a polygon that would include all the polylines of a layer in a more precise shape than a simple rectangle?

**Comment by chris3522** [comment]
Mapshaper is truly an amazing tool. Thank you for doing this and sharing.
I just discovered this feature of creating polygons online. So I was able to clip the area of ​​lines that interested me. However, I still have to extract only certain lines from among the lines. A buffer function (auto creation of a polygon surrounding a line) might be cool for this I think. In the meantime I close the issue.
![polygone_wrap](https://user-images.githubusercontent.com/60651
![polygone_wrap](https://user-images.githubusercontent.com/60651125/197555967-0ab63f1c-5702-4893-9a64-a88478f56333.png)
125/197555862-aee1f04f-6fad-432f-80f1-32d0bf21c1a7.png)
![polygone_wrap_all](https://user-images.githubusercontent.com/60651125/197555886-d6194688-ec8f-4c81-a826-2f6529fc9be5.png)
